qemu/tests/multi_nics_stress.py

```diff
@@ -116,11 +115,13 @@ def run(test, params, env):
         client_ctl = client_vm.wait_for_login(timeout=login_timeout)
         error.context("Stop fireware on netperf client guest.",
                       logging.info)
+        if params.get("pre_cmd"):
+            client_ctl.cmd_status_output(params["pre_cmd"], timeout=600)
```

Comment:
it's better to put these two lines before line 116

Reply:
Thanks for your reminder, I will place this line before 116